### DISCUSSION TITLE
ios associated-domains

### DISCUSSION THREAD

**Discussion by RiuBlackblade** [fastlane/fastlane#20831] [Q&A]
Hi,

I'm new to fastlane and most likely I'm overlooking something. I have an app with associated domains, however when i build the app with fastlane the following section doesnt get populated: ` <key>com.apple.developer.associated-domains</key><string>*</string>.`

from a fastlane perspective i tried using resign with use_app_entitlements set to true and i also tried using entitlements and pointing to the file 

I'm wondering if someone knows what am i doing wrong?

**Comment by jambstud** [comment]
we tried fastlane sigh resign    .. without any success.
Any Updates on this ? 

Building and Uploading manually the app will have the value.
